remote/remote_test.go

```diff
@@ -285,6 +282,15 @@ func testRemoteImage(t *testing.T, when spec.G, it spec.S) {
 
 		when("the image has been modified and saved", func() {
 			it("returns the new digest reference", func() {
+				var err error
+
+				img, err := remote.NewImage(
+					repoName+":some-tag",
+					authn.DefaultKeychain,
+					remote.FromBaseImage("busybox@sha256:915f390a8912e16d4beb8689720a17348f3f6d1a7b659697df850ab625ea29d5"),
+				)
+				h.AssertNil(t, err)
+
```

Comment:
Duplicate image instantiation to avoid existing complex (and incorrect) nesting.